proposals/0279-multiple-trailing-closures.md

```diff
@@ -245,6 +247,35 @@ when(2 < 3,
 )
 ```
 
+Note that "unlabeled" closures are also allowed when "empty" label placeholder `_:` is used to delimit each closure e.g.
+
+```swift
+func foo(_ fn1: () -> Void, _ fn2: () -> Void) {}
+
+foo {
+  _: { ... }
+  _: { ... }
+}
+```
+
+Calls attempting to use multiple trailing closures syntax with a single unlabeled closure are going to be transformed to use
```

Comment:
Could you use the "treated as if" terminology here, showing the proposed code and then how it maps to a call that doesn't use the trailing closure:

```swift
foo({ ... }, { ... })
```

Reply:
Sure!

Reply:
Does this look better?